ext/gmp/gmp.c

```diff
@@ -933,13 +950,12 @@ ZEND_FUNCTION(gmp_import)
 	}
 
 	if (gmp_import_export_validate(size, options, &order, &endian) == FAILURE) {
-		RETURN_FALSE;
+		RETURN_THROWS();
 	}
 
 	if ((data_len % size) != 0) {
-		php_error_docref(NULL, E_WARNING,
-			"Input length must be a multiple of word size");
-		RETURN_FALSE;
+		zend_argument_value_error(1, "must be a multiple of word size");
```

Comment:
```suggestion
		zend_argument_value_error(1, "must have length of a multiple of argument #2 ($word_size)");
```

Uh, this message starts to be a bit too complex, maybe `zend_argument_value_error()` isn't the right choice :(